forge/forge/models/config.py

```diff
@@ -10,6 +10,10 @@
 M = TypeVar("M", bound=BaseModel)
 
 
+def FromEnv(env_var: str, default: Optional[str] = None, exclude: bool = True):
+    return Field(default_factory=lambda: os.getenv(env_var, default), exclude=exclude)
```

Comment:
I'm not sure we should be treating non-default config values as default config values.

Reply:
Not sure what you mean, the value can be overwritten. The default here is only in case the env var doesn't exist. What user provides still takes precedence and that was my intention.

Reply:
I mean that we're setting `default_factory` to something that is not a real default factory, making it impossible to instantiate an actual default config object.

Reply:
I removed `FromEnv`, use `UserConfigurable` and load from env vars automatically. The code doing this is in the  `ConfigurableComponent` class.